.github/workflows/tests.yaml

```diff
@@ -82,21 +82,27 @@ jobs:
           ~/.composer/vendor/bin/php-coveralls -v
 
   functional-tests:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     env:
       SELENIUM_SERVER_DOWNLOAD_URL: http://selenium-release.storage.googleapis.com/3.141/selenium-server-standalone-3.141.59.jar
 
     strategy:
       fail-fast: false
       matrix:
+        os: ['ubuntu-latest']
         browser: ['chrome', 'firefox']
+        experimental: [false]
         selenium-server: [true, false] # Whether to run via Selenium server or directly via browser driver
         w3c: [true] # Although all builds negotiate protocol by default, it implies W3C protocol for both Chromedriver and Geckodriver
         include:
           # Force OSS (JsonWire) protocol on ChromeDriver - to make sure we keep compatibility
-          - { browser: 'chrome', selenium-server: false, w3c: false }
+          - { browser: 'chrome', os: 'ubuntu-latest', selenium-server: false, w3c: false, experimental: false }
+          - { browser: 'safari', os: 'macos-11.0', selenium-server: false, w3c: false, experimental: true }
```

Comment:
This is now failing on `No runner matching the specified labels was found: macos-11.0`. 

Also I'd prefer to remove this non-W3C build altogether. There is no value in trying to run tests, which we were not running at all until now, on deprecated versions of Safari and with deprecated OSS protocol.

Reply:
Ok, will remove it. Maybe idea was to validate if library even works with OSS SafariDriver.